tasks/copy_to_glacier/README.md

```diff
@@ -28,42 +28,91 @@ Note that this must be done for _each_ collection configured. If this list is em
 The following steps assume you are using a version of Python compliant with 3.7 (`pip` comes with current versions).
 
 ```
-python -m venv venv
+cd tasks\copy_to_glacier
```

Comment:
I think I would remove this code for building and just point them to our build script and how to run it. This would include running it in a Docker container similar to testing

Reply:
removed and replaced with the `docker run` code